include/node.h

```diff
@@ -172,6 +172,11 @@ class Node : public NodeBase<Tdim> {
   //! Apply velocity constraints
   void apply_velocity_constraints() override;
 
+  //! Assign rotation matrix
+  //! \param[in] rotation_matrix Rotation matrix of the node
+  bool assign_rotation_matrix(
```

Comment:
Why is it a `bool` if it is manually set to true for all cases?

Reply:
It will always assign, so `void` then?

Reply:
Yes